### DISCUSSION TITLE
Duda con evaluacion de predicciones

### DISCUSSION THREAD

**Discussion by cristianpenagos** [rramosp/ai4eng.v1#313] [UDEA Students]
Buenas tardes, existe algun video o guia para someter el dataset del proyecto a pruebas en kaggle, me refiero al porcentaje de aciertos.

muchas gracias

**Comment by Felipe-RA** [comment]
Saludos @cristianpenagos 

La próxima semana se va a realizar una sesión especificamente para esto, te invitamos a estar para que puedas realizar preguntas.

**Reply by cristianpenagos** [reply]
Perfecto, muchas gracias
